### ISSUE TITLE
Question about the eval_op()

### ISSUE THREAD

**Issue by Zhaoyang-Ma** [Soptq/iccv-3sfc#2]
During the validation phase, wDuring the validation phase, we typically use model.eval(), but in your eval_op() function, you're using model.train(). Is there any particular design reason for this?e typically use model.eval(), but in your eval_op() function, you're using model.train(). Is there any particular design reason for this?
![image](https://github.com/Soptq/iccv23-3sfc/assets/32972237/6ee24e1a-6b77-4130-a764-d2fbd2fb916f)

**Comment by Soptq** [comment]
@twinklexixi Ohh this is a mistake, it should be model.eval(), thanks for pointing it out!

Nevertheless, considering that no batch normalization layer is used in our models, I think this mistake should have no impact on experimental results.

I will fix this issue asap, once again thanks for your feedback!
